src/test/CheckResultDetails.test.tsx

```diff
@@ -0,0 +1,240 @@
+import React from 'react';
+import { CheckResultDetails } from '../components/CheckResultDetails';
+import { render, screen, within } from '@testing-library/react';
+import { CheckResult, CheckResultStatus } from '../types/OpsLevelData';
+
+const getCheckResult = (status: CheckResultStatus):CheckResult => ({
+  "message":`This check has status ${status}.`,
+  "warnMessage":"Something went wrong",
+  "createdAt":"2023-05-11T20:47:53.869313Z",
+  "check":{
+    "id":"1",
+    "type":"has_owner",
+    "enableOn":null,
+    "name":`Status: ${status}`,
+    "category": null,
+    owner: null
```

Comment:
```suggestion
  "check": {
    "id": "1",
    "type": "has_owner",
    "enableOn": null,
    "name": `Status: ${status}`,
    "category": null,
    owner: null
```

![eb0](https://github.com/OpsLevel/backstage-plugin/assets/2941718/254c9789-c472-49ec-b8a4-5be3a4aa17bc)

Reply:
Linting and prettier coming soon!